commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/SubmitWebRequest.kt

```diff
@@ -20,6 +20,7 @@ data class SubmitParameters(
     val attributes: List<SubmitAttribute> = emptyList(),
     val storageMode: StorageMode? = null,
     val silentMode: Boolean? = null,
+    val processAll: Boolean? = null,
```

Comment:
I would suggest `sigleJob`. The name `processAll` gives me the impression there are several submissions in the request and we're deciding whether to process them all